package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2",
-    "eslint": "^2.5.3",
-    "eslint-config-loopback": "^1.0.0"
+    "underscore": "^1.8.2"
   },
   "devDependencies": {
+    "eslint": "^2.5.3",
```

Comment:
Sounds good. @loay Can you send a PR to KB to with the 2.x.x note? Ping me over there to get that reviewed too. :shipit: ;)